cargo-shuttle/src/init.rs

```diff
@@ -1,15 +1,75 @@
+use std::fmt;
 use std::fs::{read_to_string, File};
 use std::io::Write;
 use std::path::{Path, PathBuf};
 
-use crate::args::InitArgs;
 use anyhow::Result;
 use cargo::ops::NewOptions;
 use cargo_edit::{find, get_latest_dependency, registry_url};
 use indoc::indoc;
 use toml_edit::{value, Array, Document, Item, Table};
 use url::Url;
 
+pub const FRAMEWORKS: [Framework; 9] = [
+    Framework::Axum,
+    Framework::Rocket,
+    Framework::Tide,
+    Framework::Tower,
+    Framework::Poem,
+    Framework::Salvo,
+    Framework::Serenity,
+    Framework::Warp,
+    Framework::Thruster,
+];
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub enum Framework {
```

Comment:
I'm not sure whether the distinction between `Framework` and `ShuttleInit` is really useful.

Reply:
Nah, I like this since it is the framework subset of the init command